packages/guess-webpack/src/prefetch-aot-plugin.ts

```diff
@@ -263,24 +259,20 @@ export class PrefetchAotPlugin {
       }
     });
 
-    console.log(
+    this.logger.info(
```

Comment:
Default logger level is `INFO`, so these would print still print without flag set.